venia-integration-tests/src/tests/integrationTests/multiStore/storeSwitching.spec.js

```diff
@@ -230,6 +223,61 @@ const interceptAccessoriesPageRequests = expectedStoreCode => {
     }).as('getMockFilterInput');
 };
 
+/**
+ * Setup network intercepts that return mocked data for requests
+ * specific to the tops category page
+ *
+ * @param {String} expectedStoreCode
+ */
+const interceptTopsPageRequests = expectedStoreCode => {
```

Comment:
I think we can make these intercept setup functions more dynamic if we add a second parameter for the expected category or expected product to avoid code repetition, something like: 
```suggestion
const interceptCategoryPageRequests =  (expectedCategory, expectedStoreCode) => {
     cy.intercept('GET', getCategoriesCall, getInterceptHandler(expectedCategory, expectedStoreCode)
     .as('getMockCategories');
```
and so forth. See if we can apply this to the other ones.